.stylelintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-	"extends": "stylelint-config-wordpress",
+	"extends": "@wordpress/stylelint-config",
```

Comment:
A lot of edits here are to get linting and tests to pass now that packages are updated, like `@wordpress/scripts`